src/clj/puppetlabs/services/protocols/versioned_code.clj

```diff
@@ -5,4 +5,7 @@
 
   (current-code-id
    [this environment]
-   "Returns the current code id (representing the freshest code) for the given environment."))
+   "Returns the current code id (representing the freshest code) for the given environment.")
+  (get-code-content
+   [this environment code-id file-path]
```

Comment:
So there are 2 content endpoints. static-file-content will use this endpoint and require an environment. the regular content endpoint won't need an environment and will just hit the current file content endpoint afaik. I'm probably messing the actual api endpoint names up here, but puppet will be redirecting based on whether or not the desired file is inside code dir or not.